ArduCopter/Parameters.cpp

```diff
@@ -1386,10 +1386,10 @@ void Copter::convert_pid_parameters(void)
         { "LOIT_SPEED", 3000.0f },
         { "PHLD_BRAKE_ANGLE", 800.0f },
         { "PHLD_BRAKE_RATE", 4.0f },
-        { "PSC_ACCZ_P", 0.28f },
-        { "PSC_VELXY_D", 0.0f },
-        { "PSC_VELXY_I", 0.5f },
-        { "PSC_VELXY_P", 1.0f },
+        { "PSC_ACCD_P", 0.028f },
```

Comment:
FYI @bnsgeyer, we're changing some position controller gain parameter names that will also affect tradheli